### PR TITLE
 changes in rangePicker

### PR DESCRIPTION
**Pull request by Amitrawat21**

### CODE REVIEW

File: src/components/rangepicker/RangePicker.tsx
Comment (ShubhamBade) [inline src/components/rangepicker/RangePicker.tsx:1]: please format your code

File: src/components/rangepicker/RangePicker.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  backgroundColor,
+  buttonBorderRadius,
+  color,
+  trackStyling,
+  SelectedStyling,
+  MarkerStyling,
+  
+   }) => {
+
+  const [values, setValues] = useState([20, 80]);
```
Comment (ShubhamBade) [inline src/components/rangepicker/RangePicker.tsx]: make it dynamic

File: src/components/rangepicker/RangePicker.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { View, Text } from 'react-native';
+import MultiSlider from '@ptomasroos/react-native-multi-slider';
+import {  responsiveWidth } from 'react-native-responsive-dimensions';
+import { RangePickerInterface } from '../../interface/rangePicker/RangePicker';
+
+export const RangePickerComponent :React.FC<RangePickerInterface> = ({
```
Comment (ShubhamBade) [inline src/components/rangepicker/RangePicker.tsx]: make sure do we need to set some default values

File: src/components/rangepicker/RangePicker.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+   }) => {
+
+  const [values, setValues] = useState([20, 80]);
+
+  const onValuesChange = (newValues) => {
+    setValues(newValues);
+  };
+
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
```
Comment (ShubhamBade) [inline src/components/rangepicker/RangePicker.tsx]: remove flex 1

File: src/interface/rangePicker/RangePicker.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+export interface RangePickerInterface{
+    sliderLength?: number,
+    min?: number,
+    max?: number,
+    step?: number,
+    buttonBorderRadius?: number,
+    backgroundColor?: string,
+    color?: string,
+    trackStyling?:any,
+    SelectedStyling?:any,
```
Comment (ShubhamBade) [inline src/interface/rangePicker/RangePicker.ts]: change any to object. Please update output here

File: src/components/rangepicker/RangePicker.tsx
Comment (Amitrawat21) [reply src/components/rangepicker/RangePicker.tsx:1]: hmm

**Review by rohan-arya-w — CHANGES_REQUESTED** [review]
Please resolve the comments

**Review by rohan-arya-w — CHANGES_REQUESTED** [review]
please resolve all the comments

File: src/components/rangepicker/RangePicker.tsx (hunk 3 above)
Comment (rohan-arya-w) [reply src/components/rangepicker/RangePicker.tsx]: please remove this

**Review by rohan-arya-w — APPROVED** [review]

File: src/components/rangepicker/RangePicker.tsx
Comment (Amitrawat21) [reply src/components/rangepicker/RangePicker.tsx:1]: done

File: src/components/rangepicker/RangePicker.tsx (hunk 1 above)
Comment (Amitrawat21) [reply src/components/rangepicker/RangePicker.tsx]: resolved

File: src/interface/rangePicker/RangePicker.ts (hunk 4 above)
Comment (Amitrawat21) [reply src/interface/rangePicker/RangePicker.ts]: resolved

**Review by ShubhamBade — APPROVED** [review]

**Pull request merged**